buildModels.js

```diff
@@ -0,0 +1,29 @@
+var fs = require('fs');
```

Comment:
I dunno, seems consistent to me. We've never used underscores or camelCase for any filenames. We've always used dashes. Hmm...maybe I can just come up with a name that's just one word :P